### ISSUE TITLE
Units for output 1D PDFs and stats

### ISSUE THREAD

**Issue by lea-hagen** [BEAST-Fitting/beast#222]
For the issue of PDFs for negative fluxes (#120), we wanted to display log flux values, so we're outputting the PDF/stats results for [filter]_wd_bias in symlog values.  The other fluxes are in log values.  Other fit quantities are both log and linear values.  After conversation with @karllark, we thought it might be better to just have everything saved as linear values to avoid confusion.  Before making a decision, there should be more discussion (hence creating an Issue).

**Comment by galaxyumi** [comment]
I also want to make sure that we computed the expectation values for fit parameters in log (e.g., log T, log age, etc.) in a linear space before saving as log values.

**Comment by karllark** [comment]
Good point.   Are those the only two that we want to check we are doing the computation in linear units?  Or are there more?
